test/metabase/driver_test.clj

```diff
@@ -287,6 +287,49 @@
                     :base-type         #(isa? % :type/Number)}]
                   (describe-fields-for-table (mt/db) table))))))))
 
+(deftest ^:parallel describe-fields-returns-nullability-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :test/dynamic-dataset-loading :test/create-table-without-data)
+    (mt/dataset nullable-db
+      (let [table   (t2/select-one :model/Table :id (mt/id :nullable))
+            fields  (describe-fields-for-table (mt/db) table)
+            [a b c] (->> ["a" "b" "c"]
+                         (map #(ddl.i/format-name driver/*driver* %))
+                         (map (u/index-by :name fields)))]
+        (if (driver/database-supports? driver/*driver* :describe-is-nullable (mt/db))
+          (testing ":database-is-nullable should be provided"
+            (is (= [false true false] (mapv :database-is-nullable [a b c]))))
+          (testing ":database-is-nullable should remain unspecified"
+            (is (= [nil nil nil] (mapv :database-is-nullable [a b c])))))))))
```

Comment:
Ditto, where do we test the positive cases for these features?

Reply:
Ah, found them e.g. `database-nullable-test`.

Maybe reference these in comments for each of these tests so others don't trip over this apparent gap.

Reply:
Not sure what exactly you meant by 'positive case' in this instance but I added a note to hint at the sync tests: 
cb680d9860d2b4e6d5f3732a08002758f9595692